catalog/spatial/registry/registry-admin-modules/registry-admin-remote-ui/src/main/webapp/main.js

```diff
@@ -95,7 +95,7 @@
         'marionette',
         'icanhaz',
         'js/application',
-        '../../../admin/js/HandlebarsHelpers',
+        'js/HandlebarsHelpers',
```

Comment:
I moved it into this webapp since I was starting to add functions that would only be used by the remote registry ui. I will look into what I can delete out of it.